### ISSUE TITLE
Syntax error on website

### ISSUE THREAD

**Issue by stekanius** [joomla/joomla-cms#6369]
Hello,
In the below link, under the **Integrated Help System** section there is a syntax error.
http://www.joomla.org/core-features.html
In the sentence
*" There is also a glossary explains the terms in plain English..."*
explains should be changed to **explaining**
*" There is also a glossary explaining the terms in plain English..."*

**Comment by Bakual** [comment]
@Hutchy68 Can you do that?

**Comment by wilsonge** [comment]
I have fixed it. But just want to make sure from Jess there are no other affected marketing areas using the same text as this page before closing this issue.

Thanks for reporting this @stekanius :) Really appreciated!

**Comment by jessicadunbar** [comment]
That piece of text has been on that page for years. Off hand i don't know anywhere else it's mentioned. We can close this. Thank you guys.

**Comment by zero-24** [comment]
Closing :smile: Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6369">issues.joomla.org/joomla-cms/6369</a>.</sub>
